Assets/Scripts/HeadBob.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HeadBob : MonoBehaviour
+{
+    public FpsController fpsController;
+
+    public float bobbingVerticalIntensity = 0.025f;
+    public float bobbingVerticalFrequency = 16f;
+
+    public float bobbingHorizontalIntensity = 0.015f;
+    public float bobbingHorizontalFrequency = 8f;
+
+    public float bobbingSpeed = 9f;
+
+    public float jumpBobIntensity = 0.25f;
+
+    private Vector3 _defaultLocalPosition;
```

Comment:
nit: As this is only set in Awake, this is one of those things that get buggy from hot reloading as Awake is not called after script has been reloaded and private values are not serialized.

Suggest adding something like:

```suggestion
    [SerializeField, HideInInspector]
    private Vector3 _defaultLocalPosition;
```